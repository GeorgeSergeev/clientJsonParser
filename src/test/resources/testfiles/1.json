{
	"clientId": 1,
	"subscribers": [{
		"id": 1000001,
		"spent": 1
	}]
}