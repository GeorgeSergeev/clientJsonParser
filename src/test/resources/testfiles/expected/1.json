{
  "clientId" : 1,
  "spentTotal" : 1,
  "isBig" : false
}
